components/not-mergeable-modal.tsx

```diff
@@ -43,7 +43,7 @@ export default function NotMergeableModal({
   const { activeIssue, networkIssue } = useIssue();
 
 
-  const isIssueOwner = activeIssue?.creatorGithub === user?.login;
+  const isIssueOwner = activeIssue?.creatorAddress.toLowerCase() === wallet?.address.toLowerCase();
```

Comment:
```suggestion
  const isIssueOwner = !!activeIssue && activeIssue?.creatorAddress?.toLowerCase() === wallet?.address?.toLowerCase();
```
I think without optional chaining it can throw "reading of undefined"  and we need to add a check for `activeIssue` because it can be undefined until the context loads.